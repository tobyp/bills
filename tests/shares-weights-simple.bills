#!./bills
A	B2CD3E5F.	12.0
