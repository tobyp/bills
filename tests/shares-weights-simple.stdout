todo: B pays 1.0 to A
todo: C pays 2.0 to A
todo: D pays 1.0 to A
todo: E pays 3.0 to A
todo: F pays 5.0 to A
